scripts/v3.15/php-8.1/php8-apcu/APKBUILD

```
@@ -0,0 +1,40 @@
+# Maintainer: "Diego Hernandes <iamhernandev@gmail.com>"
+
+pkgname=php8-apcu
+srcdir="/tmp/src-$pkgname"
+pkgbasedir="/tmp/pkg-$pkgname"
+_pkgreal=apcu
+pkgver=5.1.19
```

```suggestion
pkgver=5.1.21
```
https://github.com/krakjoe/apcu/issues/425
https://github.com/krakjoe/apcu/releases/tag/v5.1.21